### PR TITLE
Fixes for 64 running the platform not working on windows 10

### PR DESCRIPTION
**Pull request by barnettwilliam**
mdenet/educationplatform#64, adding gitattributes to all repositories so future commits are normalised properly across all platforms developers are using.

Updated the readme to include preconfiguration steps for windows and updated the submodules to also include the added gitattributes file.

Note - the github diff of this pull request's submodules shows changes that were previously made in addition to the actual changes. I'm not sure why this is. I did rebase some of the submodule branches to incorporate changes from merged into main to keep the working bracnh up to date. The diffs of the individual submodules pull request show the correct changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,30 @@ Any bugs, feature requests, or suggestions can be added to the [issue tracker](h
...
+``` 
+
+- Ensure that the docker daemon is running by starting Docker Desktop from the start menu or by powershell command
+``` 
+& "C:\Program Files\Docker\Docker\Docker Desktop.exe" 
+```
+
+- Ensure that the docker is set to use the linux engine by right-clicking the Docker Desktop icon in the system tray menu and selecting use linux engine or by running the command 
+```
+C:\Program Files\Docker\Docker\DockerCli.exe -SwitchDaemon -SwitchLinuxEngine
```
Comment (szschaler) [inline README.md:41]: Is `DockerCli` always installed in this location?

**Review by szschaler — CHANGES_REQUESTED** [review]
Generally LGTM. I notice that there are quite a lot of updates in the sub-modules. Is it possible that that's because these are referring to a wrong (possibly older?) SHA? Or are all these changes really needed to make things work in Windows?

File: README.md (hunk 1 above)
Comment (barnettwilliam) [reply README.md]: Not always, I have updated the readme to use DOCKER_INSTALL_PATH and mentioned the default is program files.

**Comment by barnettwilliam** [comment]
> Generally LGTM. I notice that there are quite a lot of updates in the sub-modules. Is it possible that that's because these are referring to a wrong (possibly older?) SHA? Or are all these changes really needed to make things work in Windows?

Fewer changes have been made compared to what is reported by the diff. I checked the submodule SHAs and they are correct in the commit. Its the SHA of educationplatform-docker's base submodule branch that's being used to diff against that is outdated and also showing older changes.

**Review by szschaler — APPROVED** [review]
Happy to approve assuming @arturboronat confirms it works for him with these instructions.

**Comment by arturboronat** [comment]
Hi @barnettwilliam  @szschaler,
It worked like a charm. Many thanks!

**Pull request merged**